tests/ui/from_instead_of_into.stderr

```diff
@@ -0,0 +1,78 @@
+error: Into trait is preferable than From as a generic bound
+  --> $DIR/from_instead_of_into.rs:11:5
+   |
+LL |     u32: From<T>,
+   |     ^^^^^^^^^^^^
+   |
+   = note: `-D clippy::from-instead-of-into` implied by `-D warnings`
+help: remove From bound
+   |
+LL |     ,
+   |    --
+help: Add this bound predicate
+   |
+LL |     u32: From<T>T: Into<u32>,
+   |                 ^^^^^^^^^^^^
+
+error: Into trait is preferable than From as a generic bound
+  --> $DIR/from_instead_of_into.rs:17:5
+   |
+LL |     u32: Copy + Clone + From<T>,
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+help: remove From bound
+   |
+LL |     u32: Copy + Clone,
+   |                     --
+help: Add this bound predicate
+   |
+LL |     u32: Copy + Clone + From<T>, T: Into<u32>,
```

Comment:
I think this probably should be `u32: Copy + Clone, T: Into<u32>`, but I don't know how to remove `From` showing in this span.

Reply:
You have to get the span of the `From<T>` bound and extend it to the end of the span of the `Clone` bound with `span.with_lo(clone_bound_span.hi())` And as the suggestion you can just use `String::new()`, which will automatically suggest removing it.